src/FileManager.cpp

```diff
@@ -32,16 +32,15 @@ FileManager::file_info FileManager::make_file_info(std::string filename, std::st
 std::vector<FileManager::file_info> FileManager::list_files(std::vector<std::string> extensions, bool ignore_extensions) {
   // This returns the list of files present in the folder: corePath
   // TODO: Add tests, check if corePath is not empty
-  // Converting #ifdef DEBUG and #endif to a macro
   
   std::vector<file_info> list_files;
   std::string base_name;
 
   if(*corePath.rbegin() != '/') base_name = corePath + "/";
   else base_name = corePath;
-#ifdef DEBUG
-  std::cout << "Listing files and dirs in: " << corePath << "\n";
-#endif
+
+  debugln("Listing files and dirs in:", corePath);
```

Comment:
nit: should this be: `"dirs in: "` instead of `"dirs in:` (note the extra space at the end)? Or does the variadic template adds spaces at the end every time?

Reply:
The variadic template adds a space between two variables every time.